polyfills/Element/polyfill.js

```diff
@@ -10,12 +10,15 @@
 	window.Element = window.HTMLElement = new Function('return function Element() {}')();
 
 	// generate sandboxed iframe
-	var vbody = document.appendChild(document.createElement('body'));
-	var frame = vbody.appendChild(document.createElement('iframe'));
+	var vbody = null;
+	if (!document.body) {
+		vbody = document.appendChild(document.createElement('body'));
+	}
+	var frame = (document.body||vbody).appendChild(document.createElement('iframe'));
 
 	// use sandboxed iframe to replicate Element functionality
 	var frameDocument = frame.contentWindow.document;
-	var prototype = Element.prototype = frameDocument.appendChild(frameDocument.createElement('*'));
+	var prototype = Element.prototype = (frameDocument.body||frameDocument.appendChild(frameDocument.createElement('body'))).appendChild(frameDocument.createElement('_'));
```

Comment:
If all we need is a node, can we just use the `body` node, rather than creating a new custom element under it?